pca.go

```diff
@@ -0,0 +1,76 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix"
+	"github.com/gonum/matrix/mat64"
+)
+
+// PrincipalComponents returns the k principal component direction vectors of
+// the matrix a projected score vectors and variance values by singular value
+// decomposition of the input. The input a is a matrix where variables are
```

Comment:
It would be nice to have a mathematical (or at least more in-depth) description in between the current first sentence and the description of the inputs. I even know what PCA does, and it's not clear from the documentation what "pc", "proj", and "vars" are. Such a description should at least say what the sizes of the outputs are.